### ISSUE TITLE
The "Maximize", "Maximize Left/Right" gestures require 3 swipes to maximize (left/right) a window instead of 1.

### ISSUE THREAD

**Issue by EugeneTarasenko** [aaronkollasch/jitouch#79]
The "Maximize", "Maximize Left/Right" gestures require 3 swipes to maximize (left/right) the window instead of 1.

Demonstration:
Maximize:
![maximize](https://github.com/JitouchApp/Jitouch/assets/87163666/d33f875b-aa4c-46ca-a9d6-d4ebd6b7e03f)

Maximize Left/Right
![maximise left:right](https://github.com/JitouchApp/Jitouch/assets/87163666/6ee66e24-eeac-4b34-9640-37038d672d98)

![Screenshot 2024-03-28 at 16 18 47](https://github.com/JitouchApp/Jitouch/assets/87163666/7591cb33-2fa0-49b7-ab81-87465ecd93c3)

**Comment by SuperLeiyan** [comment]
@aaronkollasch Yeah I'm having the same issue, the maximize gestures all worked instantly until recently, although for me it's only Brave Browser and VS Code, Finder seems to be fine on my end.

**Comment by learn2reid** [comment]
having this issue recently too
